kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ProxyConfigConfigMap.java

```diff
@@ -139,6 +140,14 @@ private static void addAcceptedConditions(VirtualKafkaClusterStatusFactory statu
         }
     }
 
+    private static Long getObservedGeneration(VirtualKafkaCluster cluster) {
+        ObjectMeta metadata = cluster.getMetadata();
+        if (metadata == null) {
+            return 0L;
+        }
+        return metadata.getGeneration();
```

Comment:
this looks like `ResourcesUtil#getGeneration(resource)`

Reply:
I suspect that’s me screwing up the merge. Will sort